Dshop/apps/products_catalogue/tests/test_delivery_method.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from apps.products_catalogue.models import DeliveryMethod
+
+
+@pytest.fixture
+def fixture_delivery_method():
+    return DeliveryMethod.objects.create(name='Standard', price=10.00)
+
+
+@pytest.mark.django_db
+def test_delivery_method_creation():
+    count = DeliveryMethod.objects.count()
```

Comment:
The count at  the beginning should be 0 - Django is running every test on new, empty DB. So you can simply remove this. 
You can check by 
assert 0 == DeliveryMethod.objects.count()
If this passes, remove the count.

Reply:
Test passed